pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/streams/DemoListFragment.kt

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2022. SRG SSR. All rights reserved.
+ * License information is available from the LICENSE file.
+ */
+package ch.srgssr.pillarbox.demo.ui.streams
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import androidx.lifecycle.lifecycleScope
+import androidx.navigation.fragment.findNavController
+import androidx.recyclerview.widget.DividerItemDecoration
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import ch.srgssr.pillarbox.demo.MainNavigationDirections
+import ch.srgssr.pillarbox.demo.R
+import kotlinx.coroutines.flow.collectLatest
+
+/**
+ * Demo list fragment showing a list of clickable DemoItem views.
+ *
+ * @constructor Create empty Demo list fragment
+ */
+class DemoListFragment : Fragment() {
+
+    private val viewmodel: DemoListViewModel by viewModels()
+    private lateinit var recyclerView: RecyclerView
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        // Inflate the layout for this fragment
+        return inflater.inflate(R.layout.fragment_demo_list, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        recyclerView = view.findViewById(R.id.recycler_view)
+        recyclerView.setHasFixedSize(true)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/SRGSSR/pillarbox-android/blob/9e672e88664d24e42b2fbb65a69ff4507dff46b6/pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/streams/DemoListFragment.kt#L43"></span>When using <code>setHasFixedSize() in an </code>RecyclerView<code>, </code>wrap_content<code> cannot be used as \
a value for </code>size` in the scrolling direction.</td>
    </tr>
  </tbody>
</table>